src/runner.rs

```diff
@@ -18,10 +18,10 @@ use value::{
 use {Signature, Trap, TrapKind, ValueType};
 
 /// Maximum number of bytes on the value stack.
-pub const DEFAULT_VALUE_STACK_LIMIT: usize = 1024 * 1024;
+pub const DEFAULT_VALUE_STACK_LIMIT: usize = 16 * 1024;
 
 /// Maximum number of levels on the call stack.
-pub const DEFAULT_CALL_STACK_LIMIT: usize = 64 * 1024;
+pub const DEFAULT_CALL_STACK_LIMIT: usize = 16 * 1024;
```

Comment:
Why would you like to change the defaults? I think they are reasonable for defaults...

Reply:
The initial incentive in this change is that a lot of the device don't exceed a total of 128KB of RAM.

But that change was before I added `run_start_with_stack`. I remove this from the PR.